packages/libraries/core/src/components/button/ods-button-controller.spec.ts

```diff
@@ -80,5 +80,29 @@ describe('spec:ods-button-controller', () => {
         expect(loggerSpyReferences.methodSpies.warn).toHaveBeenCalledTimes(1);
       });
     });
+    describe('methods:mutateAttributes', () => {
+      beforeEach(() => {
+        setup({
+          flex: true,
+          variant: OdsButtonVariant.flat
+        });
+      });
+
+      it('should not mutate attributes if circle attribute is false', () => {
+        controller.mutateAttributes();
+
+        expect(component.variant).toEqual(OdsButtonVariant.flat);
```

Comment:
If you're testing an exact value, you should use `toBe` instead of `toEqual`

Reply:
ok